doc/faq.rst

```diff
@@ -65,28 +65,25 @@ optimized file format such as HDF5 to reduce data load times. Various libraries
 such as H5Py, PyTables and pandas provides a Python interface for reading and
 writing data in that format.
 
-Can I add this new algorithm that I (or someone else) just published?
--------------------------------------------------------------------------
-No. As a rule we only add well-established algorithms. A rule of thumb is at least
+What are the inclusion criteria for new algorithms ?
+-----------------------------------------------------
+Generally we only add well-established algorithms. A rule of thumb is at least
 3 years since publications, 200+ citations and wide use and usefullness. A
```

Comment:
@rvraghav93  
publication is better and  
should I change the description as provided by you in the following paragraph